demo-apps/unsafe-https/README.md.gotmpl

```diff
@@ -0,0 +1,21 @@
+# dummy-ssh
+
+![Version: latest](https://img.shields.io/badge/Version-latest-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v1.0.0](https://img.shields.io/badge/AppVersion-v1.0.0-informational?style=flat-square)
+
+SSH Server for scan testing.
```

Comment:
This needs to be replaced

Would also be cool if the readme could describe **why** this demo app is considered insecure. And what to things it can be used for